.travis.yml

```diff
@@ -1,4 +1,21 @@
 sudo: required
```

Comment:
Maybe &lt;del&gt; this file too? You probably don't need both. Note that the GA one won't run on _this PR_, the workflow has to land on master before it considers running it on PRs.

Reply:
Yeah, will drop that when everything is ready.